src/virtualenv/create/creator.py

```diff
@@ -154,6 +154,15 @@ def run(self):
             safe_delete(self.dest)
         self.create()
         self.set_pyenv_cfg()
+        self._ensure_gitignore()
+
+    def _ensure_gitignore(self):
```

Comment:
On second thought this needs to be within https://github.com/pypa/virtualenv/blob/master/src/virtualenv/create/via_global_ref/api.py#L89

Reply:
And yeah please add an option to disable this via ``--no-vcs-ingore`` 👍